lib/src/main/java/com/sensirion/libble/services/sensirion/shtc1/SHTC1HistoryService.java

```diff
@@ -94,7 +94,6 @@ public SHTC1HistoryService(@NonNull final Peripheral parent, @NonNull final Blue
         mUserDataCharacteristic = super.getCharacteristic(USER_DATA_UUID);
         addCharacteristicsTo(bluetoothGattService);
         prepareCharacteristics();
-        synchronizeService();
```

Comment:
Can you elaborate: Why remove it on SHTC1 only? When is it synchronized manually then?